examples/docs_snippets/docs_snippets_tests/intro_tutorial_tests/test_type_guide.py

```diff
@@ -127,17 +132,22 @@ def double_even(even_num: EvenDagsterType) -> EvenDagsterType:
                 even_num: 2
     """
     # end_via_config
-    assert execute_solid(double_even, run_config=yaml.safe_load(yaml_doc)).success
+    assert wrap_op_in_graph_and_execute(
+        double_even, run_config=yaml.safe_load(yaml_doc), do_input_mapping=False
+    ).success
 
-    assert execute_solid(
-        double_even, run_config={"ops": {"double_even": {"inputs": {"even_num": 2}}}}
+    assert wrap_op_in_graph_and_execute(
+        double_even,
+        run_config={"ops": {"double_even": {"inputs": {"even_num": 2}}}},
+        do_input_mapping=False,
     ).success
 
     # Same same as above w/r/t chatting to prha
     with pytest.raises(AssertionError):
-        execute_solid(
+        wrap_op_in_graph_and_execute(
             double_even,
             run_config={"solids": {"double_even": {"inputs": {"even_num": 3}}}},
```

Comment:
nit: fix config?

Reply:
just about anything you see like this is fixed upstack (this is a large stack of 20ish PRs that ends in the deletion of `PipelineDefinition`/`ModeDefinition`/`PresetDefinition`)